README.md

```diff
@@ -29,50 +29,52 @@ This module requires PowerShell to be installed and the `powershell.exe` to be a
 
 The powershell module adapts the Puppet [exec](http://docs.puppetlabs.com/references/stable/type.html#exec) resource to run PowerShell commands. To get started, simply install the module and declare 'powershell' in `provider` with the applicable command.
 
-~~~
-    exec { 'RESOURCENAME':
-      command   => '$(SOMECOMMAND)',
-      provider  => powershell,
-    }
+~~~ puppet
+exec { 'RESOURCENAME':
+  command   => '$(SOMECOMMAND)',
+  provider  => powershell,
+}
 ~~~
 
 ##Usage
 
-When using `exec` resources with the `powershell` provider, the `command` parameter must be single-quoted to prevent Puppet from interpolating `$(..)`. 
+When using `exec` resources with the `powershell` provider, the `command` parameter must be single-quoted to prevent Puppet from interpolating `$(..)`.
 
 For instance, if you wanted to rename the Guest account:
 
-~~~
-    exec { 'rename-guest':
-      command   => '$(Get-WMIObject Win32_UserAccount -Filter "Name=\'guest\'").Rename("new-guest")',
-      unless    => 'if (Get-WmiObject Win32_UserAccount -Filter "Name=\'guest\'") { exit 1 }',
-      provider  => powershell,
-    }
+~~~ puppet
+exec { 'rename-guest':
+  command   => '$(Get-WMIObject Win32_UserAccount -Filter "Name=\'guest\'").Rename("new-guest")',
+  unless    => 'if (Get-WmiObject Win32_UserAccount -Filter "Name=\'guest\'") { exit 1 }',
+  provider  => powershell,
+}
 ~~~
 
 Note that the example uses the `unless` parameter to make the resource idempotent. The `command` is only executed if the Guest account does not exist, as indicated by `unless` returning 0.
 
 **Note:** PowerShell variables (e.g. `$_`), must be escaped in Puppet manifests either using backslashes or single quotes.
 
-Alternatively, you can put the PowerShell code for the `command`, `onlyif`, and `unless` parameters into separate templates and then invoke the template function in the resource.
+Alternatively, you can put the PowerShell code for the `command`, `onlyif`, and `unless` parameters into separate files and then invoke the file function in the resource. Templates and the `template()` function could also be used here if the PowerShell scripts need to have access to variables from Puppet.
 
-~~~
+~~~ puppet
 exec { 'rename-guest':
-  command   => template('guest/rename-guest.ps1'),
-  onlyif    => template('guest/guest-exists.ps1'),
+  command   => file('guest/rename-guest.ps1'),
+  onlyif    => file('guest/guest-exists.ps1'),
```

Comment:
Likely we used template because there was an issue with line endings in one of the two on Windows that was fixed at some point.